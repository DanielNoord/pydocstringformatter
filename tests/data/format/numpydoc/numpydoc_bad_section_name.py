def func():
    """Sample function with an invalid section name.

    This should raise an error.

    Exaplmes
    --------
    Run this func.

    >>> func()

    """
