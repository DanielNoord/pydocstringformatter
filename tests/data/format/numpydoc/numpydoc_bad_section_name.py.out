RuntimeError: In tests/data/format/numpydoc/numpydoc_bad_section_name.py L2-L12:

Invalid section_name 'Exaplmes'